catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,351 @@
+local id = 'CSIC/SPEI/2_8';
+local subdir = 'CSIC';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'SPEIbase: Standardised Precipitation-Evapotranspiration Index database, Version 2.8',
+  version: '2.8',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The Global SPEI database, SPEIbase, offers long-time, robust information about drought 
+    conditions at the global scale, with a 0.5 degrees spatial resolution and a monthly time resolution.
+    It has a multi-scale character, providing SPEI time-scales between 1 and 48 months. 
+    Currently it covers the period between January 1901 and December 2020.
```

Comment:
We have the time range listed in the dedicated fields, and if you add more images later, this phrase might get out of date, so I recommend deleting it.